CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 
 ## Improvements
+- PR #267 Add C++11 to Elementwise Kernels
```

Comment:
Can you expand this comment that you also replaced cupy.fftpack with cupy.fft

Reply:
Fixed